titanic/performance_soft_thresholds.json

```diff
@@ -0,0 +1 @@
+{"mse": {"baseline":0.2,"soft":0.3}}
```

Comment:
Lets revise the SDK when soft/hard tomorrow and update both the examples. Upload thresholds for all metrics and not just one.